### PR TITLE
 Update foreman-azure to 1.3.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.14
* [x] 1.13
* [x] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by mmoll** [comment]
plase rebase onto current deb/ develop

File: plugins/ruby-foreman-azure/debian/gem.list [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.3.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
 GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
 GEMS="$GEMS https://rubygems.org/downloads/azure-core-0.1.1.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-azure/debian/gem.list]: maybe these can get updated as well?

**Comment by dLobatog** [comment]
Updated - @mmoll

File: plugins/ruby-foreman-azure/debian/changelog [hunk 2]
```diff
@@ -16,6 +22,12 @@ ruby-foreman-azure (1.1.1-1) stable; urgency=low
 
  -- Daniel Lobato Garcia <me@daniellobato.me>  Tue, 15 Nov 2016 11:32:48 +0100
 
+ruby-foreman-azure (1.1.0-1) stable; urgency=low
```
Comment (domcleal) [inline plugins/ruby-foreman-azure/debian/changelog]: Not sure this should be added, this build didn't ever exist as far as I can tell. Where is it from?

Comment (dLobatog) [inline plugins/ruby-foreman-azure/debian/changelog]: It never was merged as a deb- that's true. I'll remove it.

**Comment by dLobatog** [comment]
Updated to remove 1.1.0 from the changelog

**Comment by domcleal** [comment]
Merged and building, thanks @dLobatog.

**Pull request merged**